provider/core/src/dynutil.rs

```diff
@@ -213,8 +213,6 @@ macro_rules! impl_dyn_provider {
                             })
                         }
                     )+,
-                    // Don't complain if the call site has its own wildcard match
-                    #[allow(unreachable_patterns)]
```

Comment:
Request: Can you make this an error? Experiment.

Reply:
I can't get it to work, even with `forbid`. By default it should warn anyway but it doesn't...